packages/quantic/cypress.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
     runMode: 3,
     openMode: 1,
   },
+  numTestsKeptInMemory: 0,
```

Comment:
why do we want the memory to keep 0 tests here? I think by default cypress keeps 50, was that causing the tests to be particularly slow? Won't it cause issues when tests have to re-try?

Reply:
This was a suggestion from Cypress, sometimes when the spec files have a lot of tests, the tests can crash running out of memory, one of the factors that should help avoiding this is reducing the number of tests kept in memory to 0, so I'm setting the value and I'm monitoring the tests to see if the running out of memory problem is still happening.